doc/source/rllib/rllib-checkpoints-and-exports.rst

```diff
@@ -0,0 +1,346 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. _checkpoints-and-exports-docs:
+
+#################
+Using Checkpoints
+#################
+
+
+What's a checkpoint?
+====================
+
+You can use :py:class:`~ray.air.checkpoint.Checkpoint` objects to store
+and load the current state of your :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`
+and the neural networks (weights) within.
+
+A checkpoint is a set of information, located inside a directory (which may contain
+further subdirectories) and used to restore either an :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`
+or a single :py:class:`~ray.rllib.policy.policy.Policy` instance.
+The Algorithm- or Policy instances that were used to create the checkpoint in the first place
+may or may not have been trained prior to this.
+
+RLlib uses the new Ray AIR :py:class:`~ray.air.checkpoint.Checkpoint` class to create checkpoints and
+restore objects from them.
+
+
+Algorithm- vs Policy checkpoints vs Model exports
```

Comment:
can we maybe delete this section title and move the three indicated subsections all up one level? I feel like this is unnecessarily nested. see below: 
 
![Screenshot 2022-10-07 at 13 10 55](https://user-images.githubusercontent.com/3462566/194540144-97183434-0aef-4231-bdaa-074fdbc35245.png)